src/main/java/org/fundacionjala/virtualassistant/services/mongo/RecordingServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package org.fundacionjala.virtualassistant.services.mongo;
+
+import org.fundacionjala.virtualassistant.models.mongo.Recording;
+import org.fundacionjala.virtualassistant.repository.mongo.RecordingRepo;
+import org.fundacionjala.virtualassistant.services.mongo.RecordingService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.util.List;
+
+@Service
+public class RecordingServiceImpl implements RecordingService {
+
+    @Autowired
```

Comment:
Dont use @Autowire here, spring does the injection of dependencies automatically